src/i18n.js

```diff
@@ -174,7 +174,8 @@ i18n
           projectList: {
             name: 'Project name',
             updated: 'Last updated',
-            unnamed: 'Unnamed project'
+            unnamed: 'Unnamed project',
```

Comment:
On this, I think we are defaulting the project to being called Untitled project in `defaultProjects.js` so I don't think this case will be used?

Reply:
Done in [44c9227](https://github.com/RaspberryPiFoundation/editor-ui/pull/317/commits/44c9227e01f34a99283e23b5fd69286512cfc04e)